entity-framework/efcore-and-ef6/porting/port-behavior.md

```diff
@@ -17,13 +17,15 @@ In EF6, calling `DbSet.Add()` on an entity results in a recursive search for all
 
 **EF Core performs a similar recursive search, but with some slightly different rules.**
 
-* The root entity is always in the requested state (added for `DbSet.Add` and unchanged for `DbSet.Attach`).
+* If the root entity is configured for a generated key and the key is not set, it will be put into the `Added` state.
 * **For entities that are found during the recursive search of navigation properties:**
   * **If the primary key of the entity is store generated**
     * If the primary key is not set to a value, the state is set to added. The primary key value is considered "not set" if it is assigned the CLR default value for the property type (for example, `0` for `int`, `null` for `string`, etc.).
     * If the primary key is set to a value, the state is set to unchanged.
   * If the primary key is not database generated, the entity is put in the same state as the root.
 
+For more information on these behaviors in EF Core, read [Change Tracking in EF Core](/ef/core/change-tracking/).
+
 ## Code First database initialization
 
 **EF6 has a significant amount of magic it performs around selecting the database connection and initializing the database. Some of these rules include:**
```

Comment:
I think, "is in the applications `App/Web.config` file" should be "is in the application's `App/Web.config` file."